### ISSUE TITLE
New update: Scholar Alert - Hal Daumé III - new citations

### ISSUE THREAD

**Issue by postmillenial** [postmillenial/reading-list#314]
From: scholaralerts-noreply@google.com<br>
Subject: Scholar Alert - Hal Daum&eacute; III - new citations<br>
First Attachment: http://ift.tt/19fNRe4<br>
Date: September 26, 2017 at 12:10AM<br><br><div style="font-family: arial,sans-serif; font-size: medium;">
<p><font size="2"><b>Scholar Alert: New citations to articles in Hal Daum&eacute; III's profile</b></font></p>
<h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2wgzQy5"><font size="3" color="#1A0DAB">Spectral embedding-based multiview features fusion for content-based image retrieval</font></a></h3>
<div><font size="2" color="#006621">L Feng, L Yu, H Zhu - Journal of Electronic Imaging, 2017</font></div>
<div><font size="2" color="#222222">In many computer vision applications, an object can be described by multiple features from<br>
different views. For instance, to characterize an image well, a variety of visual features is<br>
exploited to represent color, texture, and shape information and encode each feature into a<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsqSM" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2wgEEDp" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsrpO" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2wgzS9b"><font size="3" color="#1A0DAB">Model-Powered Conditional Independence Test</font></a>
</h3>
<div><font size="2" color="#006621">R Sen, AT Suresh, K Shanmugam, AG Dimakis&hellip; - arXiv preprint arXiv: &hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract: We consider the problem of non-parametric Conditional Independence testing (CI<br>
testing) for continuous random variables. Given iid samples from the joint distribution $ f (x,<br>
y, z) $ of continuous random vectors $ X, Y $ and $ Z, $ we determine whether $ X\perp Y| Z<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fn8wHu" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2wgzSGd" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fn8xew" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2wgEFar"><font size="3" color="#1A0DAB">An Attention-based Collaboration Framework for Multi-View Network Representation Learning</font></a>
</h3>
<div><font size="2" color="#006621">M Qu, J Tang, J Shang, X Ren, M Zhang, J Han - arXiv preprint arXiv:1709.06636, 2017</font></div>
<div><font size="2" color="#222222">Abstract: Learning distributed node representations in networks has been attracting<br>
increasing attention recently due to its effectiveness in a variety of applications. Existing<br>
approaches usually study networks with a single type of proximity between nodes, which<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsrWQ" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2wgEFHt" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsstS" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2wgEGev"><font size="3" color="#1A0DAB">Knowledge-based Supervision for Domain-adaptive Semantic Role Labeling</font></a>
</h3>
<div><font size="2" color="#006621">S Hartmann - 2017</font></div>
<div><font size="2" color="#222222">Semantic role labeling (SRL) is a method for the semantic analysis of texts that adds a level<br>
of semantic abstraction on top of syntactic analysis, for instance adding semantic role labels<br>
like Agent on top of syntactic functions like Subject. SRL has been shown to benefit various<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnst0U" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2wgzTKh" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsu4Y" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2wgzUhj"><font size="3" color="#1A0DAB">Integrating Specialized Bilingual Lexicons of Multiword Expressions for Domain Adaptation in Statistical Machine Translation</font></a>
</h3>
<div><font size="2" color="#006621">N Semmar, M Laib - Proceedings of PACLING 2017, 2017</font></div>
<div><font size="2" color="#222222">Abstract. Domain adaptation consists in adapting Machine Translation (MT) systems<br>
designed for one domain to work in another. Multiword expressions generally characterize<br>
specific-domains vocabularies. Translating multiword expressions is a challenge for current<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fn8zTG" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2wgzUOl" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fn8AqI" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2wgEGLx"><font size="3" color="#1A0DAB">Semi-supervised modality-dependent cross-media retrieval</font></a></h3>
<div><font size="2" color="#006621">X Dong, J Sun, P Duan, L Meng, Y Tan, W Wan, H Wu&hellip; - Multimedia Tools and &hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract In this paper, we propose a modality-dependent cross-media retrieval approach<br>
under semi-supervised conditions. The approach utilizes both labeled samples and<br>
unlabeled ones to obtain two couples of projection matrices and uses feature distance to<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsuC0" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2wgEH23" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsvWA" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2wh0lmV"><font size="3" color="#1A0DAB">DropoutDAgger: A Bayesian Approach to Safe Imitation Learning</font></a>
</h3>
<div><font size="2" color="#006621">K Menda, K Driggs-Campbell, MJ Kochenderfer - arXiv preprint arXiv:1709.06166, 2017</font></div>
<div><font size="2" color="#222222">Abstract: While imitation learning is becoming common practice in robotics, this approach<br>
often suffers from data mismatch and compounding errors. DAgger is an iterative algorithm<br>
that addresses these issues by continually aggregating training data from both the expert<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fn8BuM" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2wgR3qU" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnswK8" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2wgzVSp"><font size="3" color="#1A0DAB">Dynamic Oracle for Neural Machine Translation in Decoding Phase</font></a>
</h3>
<div><font size="2" color="#006621">ZY Dou, H Zhou, SJ Huang, XY Dai, JJ Chen - arXiv preprint arXiv:1709.06265, 2017</font></div>
<div><font size="2" color="#222222">Abstract: The past several years have witnessed the rapid progress of end-to-end Neural<br>
Machine Translation (NMT). However, there exists discrepancy between training and<br>
inference in NMT when decoding, which may lead to serious problems since the model<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsxha" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2whYdLI" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fn8C1O" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2wgzWWt"><font size="3" color="#1A0DAB">Dimensions of Semantic Similarity</font></a></h3>
<div><font size="2" color="#006621">P Szmeja, M Ganzha, M Paprzycki, W Paw&#322;owski - Advances in Data Analysis with &hellip;, 2018</font></div>
<div><font size="2" color="#222222">Abstract Semantic similarity is a broad term used to describe many tools, models and<br>
methods applied in knowledge bases, semantic graphs, text disambiguation, ontology<br>
matching and more. Because of such broad scope it is, in a &ldquo;general&rdquo; case, difficult to<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsyle" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2wgzXtv" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnsySg" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2wi3WRD"><font size="3" color="#1A0DAB">Efficient Cross-modal Retrieval Using Social Tag Information Towards Mobile Applications</font></a>
</h3>
<div><font size="2" color="#006621">J He, S Wang, Q Qu, W Zhang, Q Huang</font></div>
<div><font size="2" color="#222222">Abstract. With the prevalence of mobile devices, millions of multimedia data represented as<br>
a combination of visual, aural and textual modalities, is produced every second. To facilitate<br>
better information retrieval on mobile devices, it becomes imperative to develop efficient<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fnszWk" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2whyl2u" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2fn8CyQ" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><hr noshade size="1">
<font size="2" color="#222222">This Google Scholar Alert is brought to you by Google.<br><br><a href="http://ift.tt/2vUKjPi"><font color="#1A0DAB">Cancel alert</font></a><br><a href="http://ift.tt/2eWhNpX"><font color="#1A0DAB">List my alerts</font></a></font>
</div>
<br><br>
via Gmail
